Source/WebCore/page/scrolling/mac/ScrollerMac.mm

```diff
@@ -381,6 +381,30 @@ - (void)invalidate
         m_pair.setHorizontalScrollerImp(scrollerImp);
 }
 
+void ScrollerMac::mouseEnteredScrollbar()
+{
+    // At this time, only legacy scrollbars needs to send notifications here.
+    if (m_pair.scrollbarStyle() != WebCore::ScrollbarStyle::AlwaysVisible)
+        return;
+
+    if ([m_pair.scrollerImpPair() overlayScrollerStateIsLocked])
+        return;
+
+    [m_scrollerImp mouseEnteredScroller];
```

Comment:
Does this need to bounce to main thread?

Reply:
Fixed.